src/wishlist.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="author" content="Srujana Penugonda" />
+    <meta
+      name="description"
+      content="Awesome, budget friendly men, women and kids wear by best skilled local tailors"
+    />
+    <meta name="keywords" content="HTML, CSS, UI" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="../css/style.css" />
+    <title>Local Tailor</title>
+  </head>
+  <body>
+    <nav class="nav-container">
+      <div class="nav-body flex-dir-row">
+        <div class="nav-left-sec flex-dir-row">
+          <span
+            ><a class="text-decoration-none brand-name" href="../index.html"
+              >Local Tailor</a
+            ></span
+          >
+        </div>
+        <div class="nav-mid-sec">
+          <i class="fa fa-search search-icon"></i>
+          <input
+            type="text"
+            class="search-bar"
+            placeholder="Search for product"
+            text-large
```

Comment:
prefer using class for styling instead of attribute. custom attributes should start with `data-*` .

Reply:
Updated

Reply:
Updated